app/livechat/server/lib/Livechat.js

```diff
@@ -312,6 +312,7 @@ export const Livechat = {
 		const ret = LivechatVisitors.saveGuestById(_id, updateData);
 
 		Meteor.defer(() => {
+			Apps.getBridges().getListenerBridge().livechatEvent(AppEvents.IPostLivechatGuestSaved, _id);
```

Comment:
Prefer the shorter wrapper method

```suggestion
			Apps.triggerEvent(AppEvents.IPostLivechatGuestSaved, _id);
```